File: 10.SAMER08F.cpp
#include <iostream>
#include <string>

int main()
{
    while (true) {
        std::string line;
        std::getline(std::cin, line);
        int input = stoi(line);
        if (input == 0) {
            break;
        }
        else {
            int result = 0;
            for (int i = 1; i <= input; i++){
                result = result + i * i;
            }
            std::cout << result << "\n";
        }
    }
}

File: 10.SAMER08F.py
def no_of_squares(n):
    result = 0
    for i in range(1, n + 1, 1):
        result += pow(i, 2)
    return result

while True:
    n = int(input())
    if n == 0:
        break
    result = no_of_squares(n)
    print(result)